examples/multiplatform/xcodeproj_targets.bzl

```
@@ -22,6 +22,10 @@ XCODEPROJ_TARGETS = [
         label = "//examples/multiplatform/Tool",
         target_environments = ["device"],
     ),
+    "//examples/multiplatform/watchOSApp",
+    "//examples/multiplatform/watchOSApp/Tests/UITests",
+    "//examples/multiplatform/watchOSAppExtension",
+    "//examples/multiplatform/watchOSAppExtension/Tests/UnitTests",
```

The watch app is a dependency of `iOSApp`, and shouldn't be listed here, otherwise it might be mis-configured.

```suggestion
    "//examples/multiplatform/watchOSApp/Tests/UITests",
    "//examples/multiplatform/watchOSAppExtension/Tests/UnitTests",
```